doc/genapi.py

```diff
@@ -78,6 +79,7 @@ def genfiles(package, package_name, modules, dir='api'):
     ('kernelpy', 'Python model evaluator'),
     ('list_pars', 'Identify all parameters in all models'),
     ('mixture', 'Mixture model evaluator'),
+    #('models.__init__', 'Builtin models'),
```

Comment:
Are the built-in model docs being generated somewhere else?

Reply:
Yes. I put a comment in to let maintainers know this.